src/aoc2020/day3_copy.clj

```diff
@@ -0,0 +1,107 @@
+(ns aoc2020.day3-copy
+  (:require [aoc2020.util :refer [file->seq]]))
+
+(defn position-s
+  [slope-dist]
+  (let [row-s (take-nth (slope-dist :down) (range))
+        col-s (take-nth (slope-dist :right) (range))]
+    (map (fn [row col] [row col])
+         row-s
+         col-s)))
+
+(defn tree-at-pos?
+  [grid-template position]
```

Comment:
why isn't position destructured?

Reply:
should take a grid, not template, as an arg

Reply:
Why is this immediately jumping to a tree test? How would you break this apart?

Reply:
> Why is this immediately jumping to a tree test? How would you break this apart?

The main function is the following function `tree-count`. In `tree-count`, I generate the `pos-s` and filter the `pos-s` by `tree-at-pos?`, so this function is going to check the element at this position is `\#`

Reply:
> 

This comment would normally be in the RBC/Repl section. I'm asking you to break apart the problem.